src/site/content/en/blog/coop-coep/index.md

```diff
@@ -357,121 +355,26 @@ isolated.
-</figure>
-
-All you have to do is to put the URL indicated in the page as the reporting
-endpoint of COOP and COEP. See below to learn how to configure.
-
-#### `Report-To`
-
-To specify where the browser should send reports, append the `Report-To` HTTP
-header to any document that is served with a COEP or COOP HTTP header. The
-`Report-To` header also supports a few extra parameters to configure the
-reports. For example:
-
-```http
-Report-To: { group: 'coep_report', max_age: 86400, endpoints: [{ url: 'https://reporting-endpoint.glitch.me/post'}]},{ group: 'coop_report', max_age: 86400, endpoints: [{ url: 'https://reporting-endpoint.glitch.me/post'}]}
-```
-
-The parameters object has three properties:
-
-#### `group`
-
-The `group` property names your various reporting endpoints. Use these names to
-direct a subset of your reports. For instance, in the
-`Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy` directives you
-can specify the relevant endpoint by providing the group name to `report-to=`.
-For example:
-
-```http
-Cross-Origin-Embedder-Policy: require-corp; report-to="coep_report"
-Cross-Origin-Opener-Policy: same-origin; report-to="coop_report"
-```
-When the browser encounters this, it will cross reference the `report-to` value
-with the `group` property on the `Report-To` header to look up the endpoint.
-This example cross references `coep_report` and `coop_report` to find the
-endpoint `https://first-party-test.glitch.me/report`.
-
-If you prefer to receive reports without blocking any embedded content or
-without isolating a popup window, append `-Report-Only` to respective headers:
-i.e. `Cross-Origin-Embedder-Policy-Report-Only` and
-`Cross-Origin-Opener-Policy-Report-Only`. For example:
-
-```http
-Cross-Origin-Embedder-Policy-Report-Only: require-corp; report-to="coep_report"
-Cross-Origin-Opener-Policy-Report-Only: same-origin; report-to="coop_report"
-```
-
-By doing this, when the browser detects cross origin resources that don't have
-CORP or CORS, it sends a report using the Reporting API without actually
-blocking those resources because of COEP.
-
-Similarly, when the browser opens a cross-origin popup window, it sends a report
-without actually isolating the window because of COOP. It also reports when
-different browsing context groups try to access each other, but only in
-"report-only" mode.
-
-#### `max_age`
-
-The `max_age` property specifies the time in seconds after which unsent reports
-are to be dropped. The browser doesn't send the reports right away. Instead, it
-transmits them out-of-band whenever there aren't any other higher priority
-tasks. The `max_age` prevents the browser from sending reports that are too
-stale to be useful. For example, `max_age: 86400` means that reports older than
-twenty-four hours will not be sent.
+{% endAside %}
 
-#### `endpoints`
+To learn how to set up a server to receive reports, see [Decide where reports should
```

Comment:
Small nit, because developers need to do more than server setup:

_To learn how to configure the Reporting API and a set up a server to receive reports, head over to [Using the Reporting API](https://web.dev/reporting-api/#using-the-reporting-api)_

Or simpler

_To learn how to use the Reporting API in your site, head over to [Using the Reporting API](https://web.dev/reporting-api/#using-the-reporting-api)_